Signal/src/environment/PropertyListPreferences.m

```diff
@@ -130,9 +120,21 @@ - (void)setHasRegisteredVOIPPush:(BOOL)enabled
     [self setValueForKey:PropertyListPreferencesKeyHasRegisteredVoipPush toValue:@(enabled)];
 }
 
+- (BOOL)loggingIsEnabled
+{
+    NSNumber *preference = [NSUserDefaults.standardUserDefaults objectForKey:PropertyListPreferencesKeyEnableDebugLog];
```

Comment:
I opted for an explicit one time migration vs lazily-migrating for future code simplicity: https://github.com/WhisperSystems/Signal-iOS/pull/1636/commits/907e122d62afa0b0e612c750273a338edb35aadb